### PR TITLE
Add React example

### PR DESCRIPTION
**Pull request by crushingCodes**
Added working React Example

Sorry for all the ts-ignores, not sure how to get around them at this stage

### CODE REVIEW

File: lib/timelineRow.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
-import { TimelineRanged } from './timelineRanged';
-export interface TimelineRow extends TimelineRowStyle, TimelineRanged {
-    keyframes?: Array<TimelineKeyframe>;
-}
+import { TimelineKeyframe } from './timelineKeyframe';
+import { TimelineRowStyle } from './settings/styles/timelineRowStyle';
+import { TimelineRanged } from './timelineRanged';
+export interface TimelineRow extends TimelineRowStyle, TimelineRanged {
+    keyframes?: Array<TimelineKeyframe>;
+    title?: string;
```
Comment (crushingCodes) [inline lib/timelineRow.d.ts:6]: The title exists in the logic and in the demo, so I figured make one less ts ignore

**Comment by ievgennaida** [comment]
Thank you for this contribution. 
Unfortunately, this repository is framework agnostic so it's not proper place for the concrete implementation of the demos. 
The proper way is to have separate repositories for each realizations.

P.s. Using initialization in the functional component body is not a proper implementation due to generation of timeline objects on each render. 
The most proper way I see as implementation of a custom hook with useEffect same as did mount (to initialize timeline only once).
Also, there other changes required to make 'events' properly working.

**Comment by crushingCodes** [comment]
> The most proper way I see as implementation of a custom hook with useEffect same as did mount (to initialize timeline only once).

Ok thanks for the suggestions, I will try to use useEffect to init it, and ill try to change it to a small example.

**Comment by crushingCodes** [comment]
Hi @ievgennaida 

Ok i took your advice from above, and fixed the example by implementing useEffect(), I think the events are working correctly now and the only change for this PR ended up being the minimal example and a link to a repo I created based on your package. I dont mind if you want to clone it and update the link to your local repo, since you did most of the work. 
Here is a link for your convenience.
https://github.com/crushingCodes/animation-timeline-control-react-example

**Comment by ievgennaida** [comment]
Added integration demo to the readme

**Pull request closed without merging**